rhel7/profiles/stig.profile

```diff
@@ -254,7 +252,6 @@ selections:
     - sshd_use_priv_separation
     - sshd_disable_compression
     - chronyd_or_ntpd_set_maxpoll
-    - configure_firewalld_rate_limiting
```

Comment:
This is definitely a bug as SRG-OS-000420-GPOS-00186 references rate limiting.

Reply:
This has been explicitly removed in STIG v2r4 as can be seen in the changelog and benchmark here: https://dl.dod.cyber.mil/wp-content/uploads/stigs/zip/U_RHEL_7_V2R8_STIG.zip

`- V-72271 - Removed this requirement from the STIG.`

Should we keep the rule even if the benchmark says otherwise?

Reply:
Rate limiting is a form of Denial of Service protection internally and externally to the system. As the plan is to resubmit the RHEL7 content, we shouldn't drop rules unless it is no longer a technical feasibility e.g. dconf is used instead of gconf.